Makefile

```diff
@@ -32,7 +32,7 @@ SRC_DIRS       = $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*.go \
                    -exec dirname {} \\; | sort | uniq")
 TEST_DIRS     ?= $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*_test.go \
                    -exec dirname {} \\; | sort | uniq")
-VERSION       ?= $(shell git describe --always --abbrev=7 --dirty)
+VERSION       ?= $(shell git describe --tag --always --abbrev=7 --dirty)
```

Comment:
@simonleung8 the `--tag` needs to stay out. We should not have Git tag names in our docker image tag names unless they are being built from Travis on a tag webhook, in which case the travis build script will take care of it.